packages/cli/src/commands/generate/script/__tests__/script.test.ts

```diff
@@ -26,12 +26,19 @@ test('creates a TypeScript function to execute', () => {
   const output = files({
     name: 'typescriptyTypescript',
     typescript: true,
-  })
+  }) //?
```

Comment:
Guessing this is a wallaby artifact right?
```suggestion
  })
```